cli/command/stack/services.go

```diff
@@ -1,12 +1,17 @@
 package stack
 
 import (
+	"fmt"
+
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/command/service"
+	"github.com/docker/cli/cli/command/stack/formatter"
 	"github.com/docker/cli/cli/command/stack/kubernetes"
 	"github.com/docker/cli/cli/command/stack/options"
-	"github.com/docker/cli/cli/command/stack/swarm"
+	swarmStack "github.com/docker/cli/cli/command/stack/swarm"
```

Comment:
(see other comment below)
In other places, we used `swarmtypes` as alias for the types (instead of aliasing `stack/swarm`; would it be ok to change that? So, change this one back to:

```suggestion
	"github.com/docker/cli/cli/command/stack/swarm"
```